values/vault-secrets-webhook/vault-secrets-webhook.gotmpl

```diff
@@ -1,22 +1,13 @@
+{{- $v := .Environment.Values }}
+
 # Select namespaces where mutating admission webhook should injects (in a very non-intrusive way) the vault-env executable into containers of Deployments and StatefulSets. This executable can request secrets from Vault through special environment variable definitions.
 namespaceSelector:
   matchExpressions:
     - key: name
       operator: NotIn
       values:
-        - kube-system
-        - kube-public
-        - kube-node-lease
-        - istio-system
-        - harbor
-        - drone-pipelines
-        - keycloak
-        - knative-serving
-        - gatekeeper-system
-        - istio-operator
-        - monitoring
-        - maintenance
-        - overprovisioner
-  # matchLabels:
-  #   vault-injection: enabled
+        {{- range $v.k8s.namespaces }}
```

Comment:
can we just make it opt in? We only offer vault to `team-*` namespaces so lets only capture those.

Reply:
and are we using this from vault? I thought all secrets were coming from external-secrets...if not we can just skip this whole part, right?

Reply:
vault-secrets-webhook is not used right now. It is an option that can be enabled.

Reply:
By design we don't use it directly, so lets remove it

Reply:
ok

Reply:
done